test/remoteIncludes.spec.js

```diff
@@ -11,10 +11,10 @@ let harvester = require('../lib/harvester');
 
 let harvesterOptions = {
   adapter: 'mongodb',
-  connectionString: 'mongodb://127.0.0.1:27017/testDB',
+  connectionString: process.env.MONGODB_URL || 'mongodb://db:27017/testDB',
```

Comment:
I think thats a big ask, since it works today without it. If db is not definied you should fall back to 127.xxx or localhost. I prefer 127